features/build/metrics.feature

```diff
@@ -136,6 +136,88 @@ Feature: Builds and samples related metrics test
     Then the step should succeed
     And the output should contain:
       | cakephp-mysql-example-1  |
-      | sample-pipeline-1        |
```

Comment:
The removal of this line is intended ?

Reply:
@liangxia It‘s intended, jenkins pipeline build has no build during time record.

Reply:
Ack